### ISSUE TITLE
[Bug] MediaElement not showing up

### ISSUE THREAD

**Issue by Mikilll94** [xamarin/Xamarin.Forms#9785]
### Steps to Reproduce

1. Run this project
[MediaElementTest.zip](https://github.com/xamarin/Xamarin.Forms/files/4270913/MediaElementTest.zip)

2. As you can see MediaElement (video) is not showing.

![media_element_not_showing](https://user-images.githubusercontent.com/17653648/75614611-d485f100-5b3a-11ea-803e-8628a3cf696d.PNG)

### Expected Behavior
MediaElement (video) should be visible.

### Actual Behavior
MediaElement (video) is not visible. I was testing on Android.

### Basic Information

- Version with issue: 4.5.0.356
- IDE: 16.4.5
- Platform Target Frameworks: 
  - Android: 10

**Comment by jsuarezruiz** [comment]
I've been testing your repro sample. To have the MediaElement working, some details are missing.
The first point is the experimental flag. MediaElement is in preview and you need the use the following flag:
`Device.SetFlags(new string[] { "MediaElement_Experimental" });`
Then, if you want to auto play the video, you can use the `Autoplay` property. If you want to start playback by pressing the Play button, you need to handle the Click event and use the MediaElement Play method.

```
void Button_Clicked(object sender, System.EventArgs e)
{
     MediaPlayer.Play();
}
```

<img width="405" alt="Captura de pantalla 2020-03-02 a las 13 50 20" src="https://user-images.githubusercontent.com/6755973/75677947-d95aba00-5c8c-11ea-9752-394b388b5157.png">

**Comment by Mikilll94** [comment]
@jsuarezruiz 
You are right that I forgot to add `Device.SetFlags(new string[] { "MediaElement_Experimental" });` in Android project. After adding this I have noticed that the media player is still not displaying. Then I have changed this:

```
        <MediaElement Source="https://sec.ch9.ms/ch9/5d93/a1eab4bf-3288-4faf-81c4-294402a85d93/XamarinShow_mid.mp4"
                      ShowsPlaybackControls="True" VerticalOptions="StartAndExpand"/>
```

to this
```
        <MediaElement Source="https://sec.ch9.ms/ch9/5d93/a1eab4bf-3288-4faf-81c4-294402a85d93/XamarinShow_mid.mp4"
                      ShowsPlaybackControls="True" VerticalOptions="FillAndExpand"/>
```

So when you have `VerticalOptions="StartAndExpand"` it is not working but if change to `VerticalOptions="FillAndExpand"` it is working.

**Comment by JesperMLC** [comment]
I had the same issue now have implemented: VerticalOptions="FillAndExpand"

**Comment by Zangl** [comment]
I only see an issue with no playback when the video source is a local resource on android.

**Comment by legyas** [comment]
> I only see an issue with no playback when the video source is a local resource on android.

I have the same issue, but for some reason It works after hot reloading the page

**Comment by Gogsi** [comment]
As a temporary workaround I manually play the video and resize it to refresh the view
```
Device.StartTimer(TimeSpan.FromSeconds(1), () =>
            {
                videoPlayer.Play();

                videoPlayer.ScaleTo(0.99f);
                videoPlayer.ScaleTo(1.00f);

                return false;
            });
```

**Comment by thisisthekap** [comment]
We also can confirm this issue on Android 10 (API Level 29) with the video loaded from local resources.
